imap_processing/ultra/tests/unit/test_decom_apid_880.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+import pytest
+
+from imap_processing import decom
+
+
+@pytest.fixture()
+def decom_test_data(ccsds_path, xtce_aux_path):
+    """Read test data from file"""
+    data_packet_list = decom.decom_packets(ccsds_path, xtce_aux_path)
+    return data_packet_list
+
+
+def test_ultra_apid_880_length(decom_test_data):
+    """Test if total packets in data file is correct"""
+    total_packets = 23
+    assert len(decom_test_data) == total_packets
+
+
+def test_enumerated_apid_880(decom_test_data):
+    """Test if enumerated values derived correctly"""
```

Comment:
This is really just testing the XML definitions. We don't know that any of these paths are actually being hit because they are guarded by if-blocks.

I am not sure how much value there is here versus actually asserting against a _specific_ packet that you know about.

```python
packet = decom_test_data[0]

assert packet.data["SPINPERIODVALID"].raw_value == 0
assert packet.data["SPINPERIODVALID"].derived_value == 'INVALID'

Reply:
I agree. We can use first packet from `decom_test_data` and use that data in assert. If enums are correct in one packet, it should be correct in others as well.